gradle/compile-native-multiplatform.gradle

```diff
@@ -1,10 +1,17 @@
+import static KotlinVersion.isKotlinVersionAtLeast
+
 /*
  * Copyright 2016-2021 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
  */
 
 project.ext.nativeMainSets = []
 project.ext.nativeTestSets = []
 
+// TODO: this block should be removed when Kotlin version is updated to 1.9.20
```

Comment:
I suggest filing an issue for coroutines so we won't forget about it

Reply:
https://github.com/Kotlin/kotlinx.coroutines/issues/3828